doc/onboarding.md

```diff
@@ -203,7 +203,7 @@ Landing a PR
       * Full URL of material that might provide additional useful information or context to someone trying to understand the change set or the thinking behind it.
 * Optional: Force push the amended commit to the branch you used to open the pull request. If your branch is called `bugfix`, then the command would be `git push --force-with-lease origin master:bugfix`. When the pull request is closed, this will cause the pull request to show the purple merged status rather than the red closed status that is usually used for pull requests that weren't merged. Only do this when landing your own contributions.
 * `git push upstream master`
-    * Close the pull request with a "Landed in `<commit hash>`" comment.
+    * Close the pull request with a "Landed in `<commit hash>`" comment. If Github automatically merged your pull request then you should still add the "Landed in <commit hash>..<commit hash>" comment if you added multiple commits.
```

Comment:
I'll look at rewriting this to make it clearer, perhaps to make more of a distinction between COLLABORATOR_GUIDE and onboarding.md (https://github.com/nodejs/node/pull/9555#issuecomment-260820426).